ctk/common/src/test/kotlin/org/codice/compilance/verification/core/ResponseVerifierSpec.kt

```diff
@@ -96,23 +96,24 @@ class ResponseVerifierSpec : StringSpec() {
         }
 
         "response with incorrect InResponseTo should fail" {
-            Common.buildDom(createResponse(inResponseTo = "incorrect")).let {
+            NodeDecorator(Common.buildDom(createResponse(inResponseTo = "incorrect"))).let {
                 shouldThrow<SAMLComplianceException> {
                     ResponseVerifierTest(request, it, HTTP_POST).verify()
                 }.message?.shouldContain(SAMLCore_3_2_2_b.message)
             }
         }
 
         "response with blank version should fail" {
-            Common.buildDom(createResponse(version = " ")).let {
+            NodeDecorator(Common.buildDom(createResponse(version = " "))).let {
                 shouldThrow<SAMLComplianceException> {
                     ResponseVerifierTest(request, it, HTTP_POST).verify()
                 }.message?.shouldContain(SAMLCore_1_3_1_a.message)
             }
         }
 
         "response with non-utc instant issuer should fail" {
-            Common.buildDom(createResponse(instant = "2018-05-01T06:15:30-07:00")).let {
+            NodeDecorator(Common.buildDom(
+                    createResponse(instant = "2018-05-01T06:15:30-07:00"))).let {
                 shouldThrow<SAMLComplianceException> {
                     ResponseVerifierTest(request, it, HTTP_POST).verify()
                 }.message?.apply {
```

Comment:
✏️ You can take out the `this.` before `this.shouldContain()` in the `apply` block.